packages/frontend/src/components/ExportCSV/downloadCsv.ts

```diff
@@ -0,0 +1,12 @@
+export default function downloadCSV(url: string, filename: string = '') {
+    // Create a temporary anchor element
+    const anchor = document.createElement('a');
+    anchor.href = url;
+    anchor.download = filename;
```

Comment:
This won't work for download links coming from storage because the download attribute on a href only works for same sites. Since the server and s3 links are usually different domains, it'll redirect the page and not start a download.

We should rely on the backend to set the content disposition and not the frontend button (see #5025)

Reply:
confirming that it does not work with s3